gradle.properties

```diff
@@ -0,0 +1,13 @@
+# With JDK 16+, these JVM args are needed so that the Error Prone plugin won't use a forking compiler.
+org.gradle.jvmargs=\
```

Comment:
You should add back the default value here in addition to the new flags.

Reply:
Done, see #34